enterprise/frontend/src/embedding-sdk/lib/theme/get-embedding-theme.unit.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { getEmbeddingThemeOverride } from "./get-embedding-theme";
+
+describe("Transform Embedding Theme Override", () => {
+  it("should transform MetabaseTheme to EmbeddingThemeOverride", () => {
+    const theme = getEmbeddingThemeOverride({
+      lineHeight: 1.5,
+      fontSize: "2rem",
+      fontFamily: "Inter",
+      colors: {
+        brand: "hotpink",
+        "text-dark": "yellow",
+        "text-light": "green",
+      },
+    });
+
+    expect(theme.lineHeight).toBe(1.5);
```

Comment:
I think this is hard to read the shape of the result theme. What do you think about asserting this as an object instead?

```ts
expect(theme).toEqual({
  lineHeight: 1.5,
  fontFamily: "Inter",
  fontSizes: { md: "2rem" },
  colors: {
    brand: expect.arrayContaining(["hotpink"]),
    "text-dark": expect.arrayContaining(["yellow"]),
    "text-light": expect.arrayContaining(["green"]),
  },
});
```

Reply:
Nice! TIL that `expect.arrayContaining` is a thing 🤯